lighthouse/aggregate-service-import.md

```diff
@@ -83,55 +93,53 @@ than is possible with the DNS resolver.
 
 Here are the detailed steps for adding a `ServiceExport` to a cluster:
 
-1. When a new `ServiceExport` is created, lighthouse checks if a `ServiceImport` already exists.
-1. If not, a new `ServiceImport` is created based on the `Service` definition.
-1. If present, lighthouse checks if the `Service` definition conflicts with the `ServiceImport`.
-1. If in conflict, lighthouse sets the `ServiceExport` status to conflict.
-1. If not in conflict, lighthouse updates `ServiceImport.Status.Clusters` to include the cluster
-   name.
-1. A new `EndpointSlice` is created for the service VIP (or Pod VIPs for headless) and synced to
+1. When a new `ServiceExport` is created, a local `ServiceImport` is created containing the local `Service`
+   information.
+2. The local `ServiceImport` is checked for conflicts with the aggregated `ServiceImport` located on the broker.
+3. If in conflict, a `ServiceExportConflict` condition is set in the `ServiceExport` status and conflict resolution is
+   attempted.
+4. If a conflict cannot be resolved, the cluster's service information is not exported.
+5. Otherwise or if no conflicts, Lighthouse updates `ServiceImport.Status.Clusters` to include the cluster
```

Comment:
Will we continue to have ServiceImport in the broker namespace of the local cluster? I was not clear about that.

Reply:
Yes it will be aggregated in the broker namespace (it has to be).

Reply:
I updated the section to reflect this.